Cargo.toml

```diff
@@ -1,26 +1,22 @@
 [package]
-name = "alacritty"
+name = "alacritty-core"
 version = "0.3.0"
 authors = ["Joe Wilm <joe@jwilm.com>"]
 license = "Apache-2.0"
 build = "build.rs"
-description = "GPU-accelerated terminal emulator"
+description = "GPU-accelerated terminal emulator - library for embedders"
 readme = "README.md"
 homepage = "https://github.com/jwilm/alacritty"
 edition = "2018"
 
 [workspace]
 members = [
+    "app",
```

Comment:
I think `app` isn't great either. I feel like just calling this directory `alacritty` would be good. That should be a nice uniform name for final application and would likely make it easiest for contributors to find the main source files for the frontend.

Reply:
That sounds better to me as well.